### ISSUE TITLE
v2 -> v1: query parameters wrong encoding

### ISSUE THREAD

**Issue by cebor** [postmanlabs/postman-collection-transformer#27]
Following v2-collection-URL gets corrupted when transformed to v1-collection:

```
v2: {{baseurl}}/folder1/:uriParam/resource?foo={{foo.var}}&bar={{bar.var}}

// converts to

v1: {{baseurl}}/folder1/:uriParam/resource?foo={{foo.var}}&bar=%7B%7Bbar.var%7D%7D
```

I think all query parameters with a leading `&` are affected.

Also happens when loading a v2-collection with postman 4.4.3.

**Comment by cebor** [comment]
Any updates on this ?

**Comment by czardoz** [comment]
@cebor I'm not ale to reproduce your issue.

I took this v2 collection:

```
{
	"variables": [],
	"info": {
		"name": "collection",
		"_postman_id": "91fb7e1c-7a04-27ac-99db-97e22e1b5865",
		"description": "",
		"schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json"
	},
	"item": [
		{
			"name": "{{baseurl}}/folder1/:uriParam/resource?foo={{foo.var}}&bar={{bar.var}}",
			"request": {
				"url": {
					"raw": "{{baseurl}}/folder1/:uriParam/resource?foo={{foo.var}}&bar={{bar.var}}",
					"auth": {},
					"host": [
						"{{baseurl}}"
					],
					"path": [
						"folder1",
						":uriParam",
						"resource"
					],
					"query": [
						{
							"key": "foo",
							"value": "{{foo.var}}"
						},
						{
							"key": "bar",
							"value": "{{bar.var}}"
						}
					],
					"variable": [
						{
							"value": "",
							"id": "uriParam"
						}
					]
				},
				"method": "GET",
				"header": [],
				"body": {
					"mode": "formdata",
					"formdata": []
				},
				"description": ""
			},
			"response": []
		}
	]
}
```

And converted it to v1. The v1 collection contains the correct url:

```
{
    "id": "91fb7e1c-7a04-27ac-99db-97e22e1b5865",
    "name": "collection",
    "description": "",
    "order": [
        "74aadc8b-3127-44c1-84d0-6a9624c583f1"
    ],
    "folders": [],
    "requests": [
        {
            "id": "74aadc8b-3127-44c1-84d0-6a9624c583f1",
            "name": "{{baseurl}}/folder1/:uriParam/resource?foo={{foo.var}}&bar={{bar.var}}",
            "description": "",
            "collectionId": "91fb7e1c-7a04-27ac-99db-97e22e1b5865",
            "method": "GET",
            "headers": "",
            "dataMode": "params",
            "data": [],
            "rawModeData": "",
            "pathVariables": {
                "uriParam": ""
            },
            "url": "{{baseurl}}/folder1/:uriParam/resource?foo={{foo.var}}&bar=%7B%7Bbar.var%7D%7D",
            "responses": []
        }
    ]
}
```

**Comment by cebor** [comment]
@czardoz 

nope, the url in your example is not correct: 

`{` -> `%7B`
`}` -> `%7D`

These, chars should not get urlencoded, cause postman does not replace it, in this case.

**Comment by czardoz** [comment]
Oops, you're right, I was looking at the name. :+1:

**Comment by czardoz** [comment]
Fixed in v1.7.5

Ideally, URL encoding should completely be disabled, but I'm not sure if anything else depends on that behavior. Will track it in #30.
